de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/sharing/AnalysisSharingPresenter.java

```diff
@@ -46,10 +43,10 @@
 
     private final class LoadPermissionsCallback extends AnalysisCallback<String> {
         private final class GetUserInfoCallback implements AsyncCallback<FastMap<Collaborator>> {
-            private final List<String> usernames;
+            private final AnalysisUserPermissionsList analysisPermsList;
```

Comment:
The changes in this file mirror the changes from a refactor done in the [AppSharingPresenter](https://github.com/cyverse-de/ui/pull/59/commits/ff4511f5e19224ab8251d83804d980d51dce4933)